test/integration/script-loader/base/pages/page1.js

```diff
@@ -3,14 +3,19 @@ import Script from 'next/script'
 const Page = () => {
   return (
     <div class="container">
-      <Script
-        id="scriptBeforeInteractive"
-        src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.20/lodash.min.js?a=scriptBeforeInteractive"
-        strategy="beforeInteractive"
-      ></Script>
       <div>page1</div>
     </div>
   )
 }
 
+Page.scriptLoader = () => {
```

Comment:
Have we weighed the pros/cons of having this attached to the component instead of being a page export?

Reply:
Follow-up question here @janicklas-ralph do we want this to be `unstable_` prefixed initially so we are able to experiment with it without making breaking changes?